docs/source/about/changelog.rst

```diff
@@ -23,6 +23,14 @@ Changed:
   - :pull:`701`
 
 
+0.37.2
+------
+
+Fixed:
+
+- The boolean secure and its statement had typos, thus the wss protocol was not being called when website was served over https.
```

Comment:
```suggestion
- A typo caused IDOM to use the insecure `ws` web-socket protocol on pages loaded with `https` instead of the secure `wss` protocol - :pull:`716`
```